docs/developer/add-new-source.md

```diff
@@ -50,7 +50,24 @@ Run `sh run.sh make ../../docs/metrics/*.md` from `src/ontology`.
 ### 3.5. `docs/metrics.md`
 Add a link to your new `.md` file created in the last step.
 
-## 4. Everything else
+## 4. The `components/` goal
+When `mondo-ingest-odk.yaml` is edited and `update_repo` is run, a new `component-download-SOURCE.owl` goal is added to 
+`src/ontology/Makefile`. This essentially downloads a copy of the source. But, for it to be used in `mondo-ingest`, it 
+needs to be modified. This is done by creating a `$(COMPONENTSDIR)/omim.owl` goal in `mondo-ingest.Makefile` which runs 
+a composite `robot` command. Here are some sections you'll likely want to add that command:  
+- `rename --mappings config/SOURCE-property-map.sssom.tsv`: You'll notice that there are two other lines on most goals 
```

Comment:
## `property-map.sssom.tsv` subsection
Trish and I came to a determination in #482, and that is being codified here a bit.